client/src/linkValidator.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import LinkedModal from './linkedModal';
+import makeUrlFriendly from '../utils/makeUrlFriendly';
+
+class LinkValidator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { parklets: '', ...props };
+  }
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
Can you update to use new lifecycle hook? `componentWIllReceiveProps` will be deprecated soon

https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops

Reply:
This is for React 16.3.1 and we're on 16.2

Are we all OK with the upgrade? If so, I can make the update and update to the new lifecycle method.